### ISSUE TITLE
Battery Issues

### ISSUE THREAD

**Issue by mdigiacomi** [LineageOS-Zenfone3/android_device_asus_Z017#2]
Hi, I am having a lot of battery drain from LineageOS. "LineageOS System" is taking 27% and "Android OS" is taking up 24%. I have battery optimization on for everything possible, and I have battery saving kick in at 25%. My location services are on battery saving, and Bluetooth and WiFi are off. I probably have a screen on time for 1 to 2 hours, so I should end the day with a lot more than I have. Also, double tap to wake isn't working for me as well

**Comment by joaofl** [comment]
Me too. As I did have with the version from shakalaka. Just as described by @mdigiacomi

**Comment by mdigiacomi** [comment]
I merged the gps code from the motorola potter as it has the same chip. still throwing lots of errors though in the catlog...

**Comment by mdigiacomi** [comment]
https://builds.michaeldigiacomi.com/Z012-AND-Z017/Testing/lineage-cm-14.1-gpsfix-201709072203-UNOFFICIAL-Z017.zip

**Comment by mdigiacomi** [comment]
GPS only seems chatty with WAZE app though

**Comment by nerthazrim** [comment]
@mdigiacomi hi! does your build fix the battery issues then?

**Comment by mdigiacomi** [comment]
I don't think so. I am running it and with GPS on I'm getting 12-14 hours. I'm trying to pull the latest vendor files from the latest build. I also noticed the Moto G5 Plus is very similar so I'm trying to use some sources from there.
